src/components/sidebar/SafeList/index.tsx

```diff
@@ -75,27 +79,51 @@ const SafeList = ({ closeDrawer }: { closeDrawer?: () => void }): ReactElement =
   const hasNoSafes = Object.keys(ownedSafes).length === 0 && Object.keys(addedSafes).length === 0
   const isWelcomePage = router.pathname === AppRoutes.welcome
 
+  const handleFetchQueued = async () => {
+    const fetchedQueuedTxs: Record<string, number> = {}
+
+    Object.entries(addedSafes).forEach(([chainId, safes]) => {
+      Object.keys(safes).forEach(async (safeAddress) => {
+        const result = await getTransactionQueue(chainId, safeAddress)
+        fetchedQueuedTxs[safeAddress] = result.results.filter(isTransactionListItem).length
```

Comment:
It's fine for the safe of a prototype, but keep in mind this is paginated, so if there's `result.next`, we should show `> 10`.

Reply:
I've push a commit that handles it